docs/index.bs

```diff
@@ -3231,11 +3280,15 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
       1. Wait for |task| to have executed or for |handleFetchFailed| to be true.
       1. If |shouldSoftUpdate| is true, then [=in parallel=] run the [=Soft Update=] algorithm with |registration|.
+      1. If |raceResponseMap|[|request|] [=map/exists=], [=map/remove=] |raceResponseMap|[|request|].
```

Comment:
This step is needed to remove the map entry, which is not consumed by Fetch. The fetch handler may not have `fetch(e.request)` e.g. fallback, offline etc. Otherwise map entries are infinitely added.